### ISSUE TITLE
Empty error message for Stylint reports

### ISSUE THREAD

**Issue by NikolayFrantsev** [gucong3000/gulp-reporter#212]
cat package.json

```
{
  "dependencies": {
    "gulp": "4.0.0",
    "gulp-reporter": "2.7.0",
    "gulp-stylint": "4.0.1"
  }
}
```

cat gulpfile.js

```
const Gulp = require('gulp');
const Stylint = require('gulp-stylint');
const Reporter = require('gulp-reporter');

Gulp.task('test:stylint', () => (
  Gulp.src('packages/**/*.styl')
    .pipe(Stylint())
    .pipe(Reporter())
));
```

gulp test:stylint --silent

```
[10:25:58] packages/common-ui/src/components/ui.styl
    01:01 ❌️                                                 (StyLint)
```

**Comment by gucong3000** [comment]
@Shutnik Would you please upload `packages/common-ui/src/components/ui.styl` and `.stylintrc` here?

**Comment by NikolayFrantsev** [comment]
cat .stylintrc

```
{
  "colons": "never"
}
```

cat packages/common-ui/src/components/ui.styl

```
a
  color:red
```

**Comment by gucong3000** [comment]
@Shutnik fixed in v2.7.1
